mohawk/util.py

```
@@ -268,3 +267,28 @@ def normalize_header_attr(val):
     # Normalize like the hawk reference code.
     val = escape_header_attr(val)
     return val
+
+
+def get_bewit(resource):
+    """Returns a bewit identifier for the resource as a string"""
+    assert resource.method == 'GET'
```

this should raise a nicer exception message. I'd suggest:
````
if resource.method != 'GET':
    raise ValueError('bewits can only be generated for GET requests')
````